prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -165,6 +170,20 @@ public async Task SendRemoteUsersUpdatedAsync(Site site)
             await Send(PRIME_EMAIL, new[] { MOH_EMAIL, PRIME_SUPPORT_EMAIL }, subject, body, attachments);
         }
 
+        public async Task SendRemoteUsersNotificationAsync(Site site, IEnumerable<RemoteUser> remoteUsers)
+        {
+            var subject = "Remote Practitionner Notification";
+
+            foreach (var remoteUser in remoteUsers)
+            {
+                var body = await _razorConverterService.RenderViewToStringAsync(
```

Comment:
Doesnt look like we need to render the body each time, we can render it once and use it for each email.